pkgs/development/compilers/factor-lang/mk-factor-application.nix

```
@@ -0,0 +1,115 @@
+{ lib
+, writeText
+, makeWrapper
+, factor-lang
+, factor-no-gui
+, librsvg
+, gdk-pixbuf
+} @ initAttrs:
+
+{ name ? "${attrs.pname}-${attrs.version}"
+
+, factor-lang ? if enableUI then initAttrs.factor-lang else initAttrs.factor-no-gui
```

This change pleases `nixf-tidy` at the cost of slight inconsistence.

```suggestion
, factor-lang ? if enableUI then initAttrs.factor-lang else factor-no-gui
```
---
Changing `initAttrs.factor-lang` to `factor-lang` and do some renaming can please `nix-tify`.  Not sure if it is worth doing though.  One supports the renaming may argue that variable shadowing is bad.

---

I worked around this problem with a let binding.